tests/metaworld/envs/mujoco/sawyer_xyz/test_scripted_policies.py

```diff
@@ -0,0 +1,41 @@
+import pytest
+
+from metaworld.envs.mujoco.env_dict import ALL_ENVIRONMENTS
+from metaworld.policies import *
+from tests.metaworld.envs.mujoco.sawyer_xyz.utils import check_success
+
+
+test_data = [
+    # name,      policy,      success rate,   environment kwargs
+    ['door-open-v1', SawyerDoorOpenV1Policy(), 0.95, {}],
+    ['door-close-v1', SawyerDoorCloseV1Policy(), 0.95, {}],
+    # Expected success is low because noise has huge negative impact on drawer-open
+    ['drawer-open-v1', SawyerDrawerOpenV1Policy(), 0.50, {}],
```

Comment:
perhaps make action_noise_std part of these params so that you can customize for environments based on their sensitivity.

this also means you could run a non-noisy version of this test to verify it's 100% solvable with deterministic actions (please do that, actually)